Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -614,8 +614,13 @@ public class JavaScriptLifter: Lifter {
                 w.decreaseIndentionLevel()
                 w.emit("}")
 
-            case is BeginForOf:
-                w.emit("for (\(decl(instr.innerOutput)) of \(input(0))) {")
+            case let op as BeginForOf:
+                if op.indices.count > 1 {
```

Comment:
Is this correct? What if that index is != 0?

Reply:
This was incorrect and should now be fixed with the latest commit (i.e. two unique ops for for-of loops).
I've also included this as part of the lifting test case.